packages/db/src/types/record.ts

```diff
@@ -0,0 +1,4 @@
+export interface updatedAtRange {
+  updatedAtFrom?: Date | null,
```

Comment:
Is `| null` necessary with the `?` operator?

Reply:
96eea93dc